src/messages/MFSMap.h

```diff
@@ -48,11 +50,11 @@ class MFSMap : public Message {
   void decode_payload() {
     bufferlist::iterator p = payload.begin();
     ::decode(epoch, p);
-    ::decode(encoded, p);
+    ::decode(fsmap, p);
   }
   void encode_payload(uint64_t features) {
     ::encode(epoch, payload);
-    ::encode(encoded, payload);
+    ::encode(fsmap, payload, features);
```

Comment:
I didn't think passing explicit features arguments (vs. how it was before) to ::encode actually changed the encoding on the wire, does it?

Obviously changing the actual feature bits changes things, but this is just changing it to pass through the globally configured features as opposed to having a hardcoded set further down.